ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -162,7 +162,6 @@ tests:
     cluster_profile: aws-qe
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:oldest-supported-z-rollback
-      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
```

Comment:
@jianlinliu  If we make those changes in `config/openshift/release` in another PR, we can self review/approve this PR to get the other changes in.